image/tarexport/save.go

```diff
@@ -394,10 +397,12 @@ func (s *saveSession) saveLayer(id layer.ChainID, legacyImg image.V1Image, creat
 			return distribution.Descriptor{}, err
 		}
 
-		for _, fname := range []string{"", legacyVersionFileName, legacyConfigFileName, legacyLayerFileName} {
-			// todo: maybe save layer created timestamp?
-			if err := system.Chtimes(filepath.Join(outDir, fname), createdTime, createdTime); err != nil {
-				return distribution.Descriptor{}, err
+		if createdTime != nil {
+			for _, fname := range []string{"", legacyVersionFileName, legacyConfigFileName, legacyLayerFileName} {
+				// todo: maybe save layer created timestamp?
+				if err := system.Chtimes(filepath.Join(outDir, fname), *createdTime, *createdTime); err != nil {
```

Comment:
See my previous comment (haven't given it a lot of thought, so add pinches of salt)

Minor nit (in the smallest possible "minor"): I think we mostly use TODO (all-caps);

```bash
git grep 'TODO' | wc -l
1818

git grep 'todo' | wc -l
63
```


```suggestion
		if createdTime != nil {
			for _, fname := range []string{"", legacyVersionFileName, legacyConfigFileName, legacyLayerFileName} {
				// TODO: maybe save layer created timestamp?
				if err := system.Chtimes(filepath.Join(outDir, fname), *createdTime, *createdTime); err != nil {
```

Reply:
This isn't my comment :sweat_smile:  https://github.com/moby/moby/pull/45504/files/c5f65f38625799641c8044852339491019cf4e9f#diff-9ac2cf7826d0cfc8316ca97dceae47447f431bc510fd4a4622ebeb53cc74d08aL398

(if you review without whitespace, this isn't even a modified line :see_no_evil:)

Reply:
DOH! Not sure why I didn't spot that 🙈